exercises-latex/example-lab-report/loesung/v16516/latex_formatting.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+from uncertainties.unumpy import (
+    std_devs as stds,
+)
+
+def make_qty(num, unit, exp="", figures=None):
+    """Format an uncertainties ufloat as a \qty quantity"""
+    if np.any(stds([num])):
+        if figures is None:
+            figures = ""
+        x = "{0:.{1:}uf}".format(num, figures).replace("/", "")
```

Comment:
I would always choose for f-strings at this point and I think they are more "modern" and [here](https://realpython.com/python-f-strings/) they claim they are faster than `.format`